### ISSUE TITLE
[Feature Request]  Migration to Plugin?

### ISSUE THREAD

**Issue by EldrinSMP** [elkorol/Stash-App-Userscript-Tagger-View-Div-Remover#1]
With the latest release, userscripts have been migrated to plugins in StashApp. This is one of my MUST HAVE userscripts, and I would love to see is back in Stash. Trying to use the script in Violent.Tampermonkey now gives a syntax error.
/

**Comment by elkorol** [comment]
Thanks. I'll port it but I'm not sure where it belongs. I'm not sure if it will be accepted in the community repo. It was made to compliment 7dJx1qP's userscript batch save. He has his own repo, with a source [yml](https://7djx1qp.github.io/stash-plugins/main/index.yml) to add to Stash, so it really belongs in there. But I'm not sure if he accepts pull requests. I'll ask him on discord.

**Comment by elkorol** [comment]
A better version has been added to the batch save userscript in this repo, [7dJx1qP stash-userscripts](https://github.com/7dJx1qP/stash-userscripts)

**Comment by EldrinSMP** [comment]
> Thanks. I'll port it but I'm not sure where it belongs. I'm not sure if it will be accepted in the community repo. It was made to compliment 7dJx1qP's userscript batch save. He has his own repo, with a source [yml](https://7djx1qp.github.io/stash-plugins/main/index.yml) to add to Stash, so it really belongs in there. But I'm not sure if he accepts pull requests. I'll ask him on discord.

Hopefully you can find a place for it. It really is a huge part of my Stash setup that I am HATING doing without..


> A better version has been added to the batch save userscript in this repo, [7dJx1qP stash-userscripts](https://github.com/7dJx1qP/stash-userscripts)

I actually checked that out before coming here, but I don't see any plugins or userscripts there that include the remove scene functionality.

**Comment by elkorol** [comment]
It's just been added to the batch save user script since it was made to compliment that function in the first place

**Comment by EldrinSMP** [comment]
> It's just been added to the batch save user script since it was made to compliment that function in the first place

Makes sense. Has that userscript been converted over to a plugin yet?  LOL,
